src/c++/library/grpc_client.cc

```diff
@@ -310,6 +312,16 @@ InferResultGrpc::RawData(
   return Error::Success;
 }
 
+Error
+InferResultGrpc::IsFinalResponse(bool* is_final_response) const
+{
+  if (is_final_response == nullptr) {
```

Comment:
Any reason this can't be a reference instead of a pointer?

Reply:
It's the pattern for the other getters.